Documentation/disaster-recovery.md

```diff
@@ -196,19 +196,21 @@ The following `kubectl patch` command is an easy way to do that. In the end it p
 
 ```console
 mon_host=$(kubectl -n rook-ceph get svc rook-ceph-mon-b -o jsonpath='{.spec.clusterIP}')
-kubectl -n rook-ceph edit secret rook-ceph-config -p '{"stringData": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "'"${good_mon_id}"'"}}'
+kubectl -n rook-ceph patch secret rook-ceph-config -p '{"stringData": {"mon_host": "[v2:'"${mon_host}"':3300,v1:'"${mon_host}"':6789]", "mon_initial_members": "'"${good_mon_id}"'"}}'
 ```
 
 > **NOTE**: If you are using `hostNetwork: true`, you need to replace the `mon_host` var with the node IP the mon is pinned to (`nodeSelector`). This is because there is no `rook-ceph-mon-*` service created in that "mode".
 
 ### Restart the mon
 
-You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl replace` on the backup of the mon deployment yaml:
+You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl delete` to restart the pod. :
```

Comment:
```suggestion
You will need to "restart" the good mon pod to pick up the changes. For this run `kubectl delete` to restart the pod:
```

Reply:
Fixed

Reply:
You don't need a commit for this since you introduced that typo. Just amend the commit that introduced it.